src/Shared/PackageManagers/NuGetProjectManager.cs

```diff
@@ -221,9 +248,10 @@ private async Task UpdateVersionMetadata(PackageMetadata metadata, NuGetPackageV
             }
 
             string nameLowercase = packageVersionPackageVersionMetadata.Name.ToLowerInvariant();
+            string versionLowercase = metadata.PackageVersion.ToLowerInvariant();
 
             // Set the version specific URI values.
-            metadata.VersionUri = $"{metadata.PackageManagerUri}/packages/{nameLowercase}/{metadata.PackageVersion}";
+            metadata.VersionUri = $"{metadata.PackageManagerUri}/packages/{nameLowercase}/{versionLowercase}";
```

Comment:
The metadata reporting URI ends up differing from the actual URI used for the query which is not lowercased.